configs.py

```diff
@@ -9,20 +9,15 @@ def load_version():
   return package.get("name") + " v" + package.get("version")
 
 # Load price feeds configuration parameters from file
-def load_price_feeds_config(filename, network_name):
-  config = json.load(open(filename))
+def load_price_feeds_config(path, network_name):
   chain_name = network_name.split('.')[0]
+  try:
+    if path.startswith("http"):
+      http = urllib3.PoolManager(timeout=3.0)
+      response = http.request('GET', path)
+      config = json.loads(response.data.decode('utf-8'))
+    else:
+      config = json.load(open(path))
+  except Exception as ex:
+    return None 
```

Comment:
I'd expect any exceptions to be logged at his point, specially to avoid overlooking misconfigured URLs or paths:

```suggestion
    print(ex)
    return None 
```